### PR TITLE
Added accumulator/counters for frame and bytes written/read on Connect/ConnectReply

### PR DESCRIPTION
**Pull request by dpwspoon**
depends on/includes https://github.com/reaktivity/nukleus-sse.java/pull/8

### CODE REVIEW

**Comment by dpwspoon** [comment]
requires https://github.com/reaktivity/reaktor.java/pull/29

File: src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java [hunk 1]
```diff
@@ -632,7 +691,8 @@ private void doSseBegin(
         long streamRef,
         long correlationId,
         String pathInfo,
-        String lastEventId)
+        String lastEventId
+        )
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java]: Please revert this change.

**Review by jfallows — APPROVED** [review]

**Review by jfallows — CHANGES_REQUESTED** [review]
Please update formatting issue before we merge.

**Review by jfallows — DISMISSED** [review]

**Review by jfallows — APPROVED** [review]
